homeassistant/components/backup/manager.py

```diff
@@ -1726,7 +1726,7 @@ def is_excluded_by_filter(path: PurePath) -> bool:
             """Filter to filter excludes."""
 
             for exclude in excludes:
-                if not path.match(exclude):
+                if not path.full_match(f"data/{exclude}"):
```

Comment:
Where does `"data/"` come from?

Reply:
It's what we name the core config directory when adding it to the tar here: https://github.com/home-assistant/core/blob/162804b78598ad4f6a0496f6060f57a40311750f/homeassistant/components/backup/manager.py#L1751-L1756

Reply:
I'd add a comment about that.

Reply:
I added a comment to make it clearer